partner_type_base/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013 Nicolas Bessi (Camptocamp SA)
+# Copyright 2014 Agile Business Group (<http://www.agilebg.com>)
+# Copyright 2015 Grupo ESOC (<http://www.grupoesoc.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner Address Types",
+    "summary": "Base implementation to improve the address type customization.",
+    "version": "18.0.1.0.0",
+    "author": "glueckkanja AG, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "maintainer": "CRogos",
+    "category": "Extra Tools",
+    "website": "https://github.com/OCA/partner-contact",
+    "depends": ["base_setup"],
```

Comment:
Why not `base` ?

Reply:
Good question... I've copied it from "partner_firstname" module. 

There is a module in odoo/addons/base_setup which depends on "base" and "web". It also has some minor changes on the patner kanban view. So maybe a view order issue?

A general question: 
If partner_firstname has a dependency to partner_type_base and partner_type_base depends on base, but partner_firstname also references base in a view. Shall partner_firstname reference base explicit in the manifest or should it not add the base to the depends?

partner_firstname.depends = ["partner_type_base"] or ["base", "partner_type_base"] ???
when
partner_type_base.depends = ["base"]